# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x86/apmd/skip_xapm.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2002-07-25 08:37:45.000000000 +0200
+++ ./Makefile	2002-07-25 08:39:39.000000000 +0200
@@ -28,7 +28,10 @@
 .SUFFIXES:
 
 OBJS=apmlib.o
-EXES=apm apmd xapm apmsleep
+EXES=apm apmd apmsleep
+ifndef SKIP_XAPM
+  EXES += xapm
+endif
 HEADERS=apm.h
 
 PREFIX=/usr
@@ -94,8 +97,10 @@
 	install        apmd            ${PREFIX}/sbin
 	install -b -Vt apmd_proxy      ${APMD_PROXY_DIR}
 	install -m 644 apmd.8          ${MANDIR}/man8
+ifndef SKIP_XAPM
 	install        xapm            ${PREFIX}/bin
 	install -m 644 xapm.1          ${MANDIR}/man1/xapm.1
+endif
 	install        apmsleep	       ${PREFIX}/bin
 	install -m 644 apmsleep.1      ${MANDIR}/man1
 	install -m 644 apmsleep.1.fr   ${MANDIR}/fr/man1
